indico/modules/events/api.py

```diff
@@ -147,7 +150,248 @@ def export_event(self, aw):
+                '_type': person_type,
+                '_fossil': self.fossils_mapping['person'].get(person_type, None),
+                'fullName': person.get_full_name(last_name_upper=False, abbrev_first_name=False),
+                'id': unicode(person.id),
+                'affiliation': person.affiliation,
+                'emailHash': md5(person.email).hexdigest() if person.email else None
+            }
+            if isinstance(person, PersonLinkBase):
+                data['db_id'] = person.id
+                data['person_id'] = person.person_id
+            if self.user and can_manage:
+                data['email'] = person.email or None
+            if person_type == 'ConferenceChair':
+                data['fullName'] = person.get_full_name(last_name_upper=False, abbrev_first_name=False,
+                                                        show_title=True),
+            return data
+
+    def _serialize_persons(self, persons, person_type, can_manage=False):
+        return [self._serialize_person(person, person_type, can_manage) for person in persons]
+
+    def _serialize_convener(self, convener, can_manage=False):
+        data = {
+            'fax': '',
+            'familyName': convener.last_name,
+            'name': convener.get_full_name(last_name_upper=False, abbrev_first_name=False),
+            'firstName': convener.first_name,
+            'title': convener.title,
+            '_type': 'SlotChair',
+            'affiliation': convener.affiliation,
+            '_fossil': 'conferenceParticipation',
+            'fullName': convener.get_full_name(last_name_upper=False, abbrev_first_name=False),
+            'id': convener.person_id,
+            'emailHash': md5(convener.person.email).hexdigest() if convener.person.email else None
+        }
+        if can_manage:
+            data['address'] = convener.address,
+            data['phone'] = convener.phone,
+            data['email'] = convener.person.email,
+        return data
+
+    def _serialize_session(self, session_, can_manage=False):
+        return {
+            'folders': build_folders_api_data(session_),
+            'startDate': self._serialize_date(session_.start_dt) if session_.blocks else None,
+            'endDate': self._serialize_date(session_.end_dt) if session_.blocks else None,
+            '_type': 'Session',
+            'sessionConveners': [self._serialize_convener(c, can_manage) for c in session_.conveners],
+            'title': session_.title,
+            'color': '#{}'.format(session_.colors.background),
+            'textColor': '#{}'.format(session_.colors.text),
+            'description': session_.description,
+            'material': build_material_legacy_api_data(session_),
+            'isPoster': session_.is_poster,
+            'url': url_for('sessions.display_session', session_, _external=True),
+            'protectionURL': url_for('sessions.session_protection', session_.event_new, session_, _external=True),
```

Comment:
useless (only works fine in ajaxdialog) and was never intended to be in the API (added in 8037867b) -> remove